test/src/edu/stanford/nlp/simple/TokenTest.java

```diff
@@ -112,4 +113,25 @@ public void testEndPosition() {
 
     assertEquals(35, token.endPosition());
   }
+
```

Comment:
While this is technically correct, I think it would be more useful to put it in SentenceTest and check the after() of various values between 0 and n-1 as well